install_config/syncing_groups_with_ldap.adoc

```diff
@@ -433,6 +433,153 @@ users:
 <1> The name of the Group as specified by the user-defined name mapping.
 ====
 
+[[rfc2307-with-error-tolerances]]
+=== RFC 2307 with User-Defined Error Tolerances
+
```

Comment:
Probably too much detail. Just two paragraphs, one for each error type, include the actual error text (will help troubleshooting searches find this section), a short description of what causes it, and mention the config field name.